src/azure-cli/azure/cli/command_modules/netappfiles/_params.py

```diff
@@ -85,7 +85,10 @@ def load_volumeArguments(self, account_name_type, pool_name_type, volume_name_ty
         c.argument('policy_enforced', arg_type=get_three_state_flag())
         c.argument('kerberos_enabled', arg_type=get_three_state_flag())
         c.argument('smb_encryption', arg_type=get_three_state_flag())
-        c.argument('smb_continuously_available', options_list=['--smb-continuously-avl'], arg_type=get_three_state_flag())
+        c.argument('smb_continuously_avl', options_list=['--smb-continuously-avl'], arg_type=get_three_state_flag())
```

Comment:
It's unnecessary to add options_list as `smb_continuously_avl` will be exposed as an argument `--smb-continuously-avl` to CLI customers.

Reply:
options_list has been removed